pysql_beam/sql_io/sql.py

```diff
@@ -380,6 +406,20 @@ def _create_connection(self):
                                            user=self.username, password=self.password,
                                            database=self.database)
             _connection.autocommit = self.autocommit or AUTO_COMMIT
+        elif self.wrapper == MSSQLWrapper:
+            print('host', self.host)
+            print('port', self.port)
+            print('database', self.database)
+            print('username', self.username)
+            print('password', self.password)
```

Comment:
Please do not log user name and password, sometimes these being end up logged in log files